### PR TITLE
Snapshot test

### PR DESCRIPTION
**Pull request by Victoria129**
- [ ] I have added a snapshot tests file to tests.

### CODE REVIEW

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/extensions
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```
Comment (omar-muhamad) [inline src/__tests__/calculate.test.js]: - [ ] Please remove eslint disablement line and try to fix linters errors yourself. 
[hint:] I removed this line and there is no error found.

- [ ] Please remove any commented(unused) code like line 3 to make your code clean.

File: src/__tests__/snapshottests.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+describe('Snapshot for Calculator and Home', () => {
+  it('Snapshot should match component render', () => {
+    const homeview = render(<Home />);
+    expect(homeview).toMatchSnapshot();
+  });
+  it('Snapshot should match component render', () => {
+    const quoteview = render(<Quote />);
+    expect(quoteview).toMatchSnapshot();
+  });
```
Comment (omar-muhamad) [inline src/__tests__/snapshottests.test.js:16]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=js%20using%20Jest.-,Create%20unit%20tests%20for%20all%20React%20components%3A,Use%20Jest%20snapshots%20to%20test%20the%20components.,-Use%20React%20Testing), Please add screenshot tests for the `Calculator` and `navbar` components as you did with `home` and `quote` components.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Victoria129 and @gsmalik030,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/__tests__/snapshottests.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  it('Snapshot should match component render', () => {
+    const navview = render(<Navbar />);
```
Comment (youmari) [inline src/__tests__/snapshottests.test.js]: - [ ] Nice adding the required changes by the previous reviewer however  one of the tests is failing please make sure to fix it 
Screenshot 2023-02-22 at 15 14 09

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Victoria129 @gsmalik030 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 
✔️ Clean code 
✔️ PRO PR

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Victoria129  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- All tests are passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**